coforma/swift-tech-challenge
# [59] Add Mixpanel analytics

## Summary

Mixpanel is a customer and product analytics platform that we use to track analytics related to traffic and user behavior. This helps us better meet user needs by monitoring application usage so we can improve product performance and behavior.

It's best practice to separate your tracked data by environment. We have two separate Mixpanel projects set up: [Test](https://mixpanel.com/project/3250005/view/3758561/app/boards#discover) (used for testing conducted in lower environments) and [Prod](https://mixpanel.com/project/3250083/view/3758651/app/boards) (used for tracking production data). This data can be seen after invitation and authentication. For current tech challenge team members, you can authenticate with Google SSO.

Closes #59 (issue)

## Type of change

## How To Test
1. Log in to Mixpanel test project (see above). Click 'Events'.
2. Click on 'Apply to this school' button on main page.
3. View your events in Mixpanel (note: adblockers may block this -- best to try in incognito).

## Checklist:
- ~[] If there are security concerns they are addressed or ticketed after being discussed~
